src/main/java/com/teamscale/upload/utils/OkHttpUtils.java

```diff
@@ -60,10 +63,27 @@ public class OkHttpUtils {
 	 * @param trustStorePassword
 	 *            May be null if no trust store should be used.
 	 */
-	public static OkHttpClient createClient(boolean validateSsl, String trustStorePath, String trustStorePassword,
+	public static OkHttpClient createClient(boolean validateSsl, String proxyHost, String trustStorePath, String trustStorePassword,
 											long timeoutInSeconds) {
 		OkHttpClient.Builder builder = new OkHttpClient.Builder();
+		if(proxyHost != null) {
+			String[] proxyHostParts = proxyHost.split(":");
+			if(proxyHostParts.length == 3) {
+				String host = proxyHostParts[1].replace("/", "");
+				String port = proxyHostParts[2];
+
+				Proxy proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress(host, Integer.parseInt(port)));
```

Comment:
Should we try-catch the error in case the port is not a number or do we just let it fail then?